src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -915,6 +916,11 @@ public static string SimulationDoesNotHaveOutputPath(string simulationName, stri
          {
             return $"Simulation {simulationName} does not have the output path {outputPath}";
          }
+
+         public static string ValueUpdatedFrom(string parameterIdentification, string isoDate)
+         {
+            return $"Value updated from '{parameterIdentification}' on {isoDate}";
```

Comment:
That text may have to be revised. I just want to have the function in at the moment